sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosContainer.java

```diff
@@ -799,7 +792,6 @@ public void enableGlobalThroughputControlGroup(ThroughputControlGroupConfig grou
      *  </p>
      *
      */
-    @Beta(value = Beta.SinceVersion.V4_14_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
     public void openConnectionsAndInitCaches() {
```

Comment:
@xinlian12 - are we ready to GA this? Or are there going to be any more behavior breaking changes?

Reply:
We have it in beta now for over a year and recommended it to several customers - no way that we can make any breaking changes.